pandora-client-web/src/components/chatroomAdmin/chatroomAdmin.tsx

```diff
@@ -507,18 +534,18 @@ function BackgroundSelectDialog({ hide, current, select }: {
 
 	const availableBackgrounds = useMemo(() => assetManager.getBackgrounds(), [assetManager]);
 	const [nameFilter, setNameFilter] = useState('');
-	/*
-	 * TODO: Add a tag based filter to the dialog in a later version
-	 * const [tagFilter, setTagFilter] = useState('');
-	 */
+	const [tags, setTags] = useState<readonly string[]>([...assetManager.backgroundTags.keys()]);
+	const knownTagCategories = useMemo(() => uniq([...assetManager.backgroundTags.values()].map((tag) => tag.category)), [assetManager.backgroundTags]);
```

Comment:
Selection behavior:
Having all selected by default and unselecting feels very weird - one has to first unselect before selecting those that are wanted.
We think much better would be to have by default all unselected and have a special case that none selected is the same as all selected (empty filter is ignored)